frontend/www/js/omegaup/components/course/Statistics.vue

```diff
@@ -295,6 +295,7 @@ export default class Statistics extends Vue {
     return {
       chart: {
         type: 'bar',
+        height: `${data.length < 10 ? null : (data.length * 10) / 2}%`,
```

Comment:
se puede hacer que esto no despliegue `null%`?

```suggestion
        height: data.length < 10 ? null : `${data.length * 5}%`,
```

también, por qué el cambio de comportamiento? no sería mejor mantener el comportamiento consistente sin importar la cantidad de estudiantes? también, qué pasa si hay más de 20 estudiantes?

Reply:
Creo que se hace muy pequeña la grafica si solo hay 1 o 2 problemas y con el tamaño que tiene al poner `null` se ve mejor, no sé si se refiera a eso en el cambio de comportamiento, la grafica puede crecer más del 100% y puede seguir creciendo sin importar la cantidad de problemas.